include/stdlib.h

```diff
@@ -128,6 +128,15 @@ extern "C"
 
 void      srand(unsigned int seed);
 int       rand(void);
+double    drand48(void);
+double    erand48(unsigned short xsubi[3]);
```

Comment:
Should we wrap this with `#ifdef CONFIG_HAVE_DOUBLE`?

Reply:
Done!